drivers/adodb-pdo.inc.php

```diff
@@ -64,79 +64,97 @@ function adodb_pdo_type($t)
 
 /*----------------------------------------------------------------------------*/
 
-
-class ADODB_pdo extends ADOConnection {
+class ADODB_pdo extends ADOConnection 
```

Comment:
Shouldn't this be an *abstract* class now ? If I'm not mistaken after the refactoring it's only ever used as a base class for the PDO drivers.

As a nice side-effect, this would ensure legacy `ADONewConnection('pdo')` calls are no longer possible, as a _PHP Fatal error:  Uncaught Error: Cannot instantiate abstract class ADODB_pdo_ exception would be thrown in this case.